Silicon/NXP/iMX6Pkg/Library/iMX6ClkPwrLib/iMX6ClkPwr.c

```diff
@@ -1880,3 +1880,4 @@ ImxClkPwrGetClockInfo (
 {
   return ImxpGetClockInfo (&mImxpClockPwrCache, ClockId, ClockInfo);
 }
+

```

Comment:
No need for this extra newline, it can be removed to remove this file from this commit.